release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -100,6 +100,12 @@ For more information, see xref:../installing/installing_aws/installing-aws-secre
 
 //For more information, see ../installing/installing_nutanix/installing-nutanix-installer-provisioned.adoc#installing-nutanix-installer-provisioned[Installing a cluster on Nutanix].
 
+[id="ocp-4-11-capabilities"]
+==== Additional customization when installing {product-title}
+{product-title} 4.11 allows you to disable the installation of the `metal3` API, the optional `redhat-marketplace` Operators, and the `samples` content that is stored in the `openshift` namespace. You can disable these features by setting the `baselineCapabilitySet` and `additionalEnabledCapabilities` parameters in the `install-config.yaml` configuration file prior to installation.
```

Comment:
`redhat-marketplace` looks like one optional operator catalog for marketplace, see doc [Red Hat-provided Operator catalogs](https://docs.openshift.com/container-platform/4.10/operators/understanding/olm-rh-catalogs.html). Maybe it is better to describe three operator names instead of its function here -  baremetal, marketplace, and openshift-smaples, which also are the valid value for `additionalEnabledCapabilities`. 
cc: @jianzhangbjz if more comments.

Reply:
I'm not sure if the `metal3 API` and the `samples content` are the proper expression. So involving @pamoedom @tisutisu to take a look.